public/locales/en/common.json

```diff
@@ -363,6 +363,7 @@
   },
   "table": {
     "mints": "Mints",
+    "view": "View",
```

Comment:
if we add something to the common.json or any already existing language files, that should be added to all language files. 
adding an English version to all languages is fine, except Russian